### PR TITLE
Azure pipelines

### PR DESCRIPTION
**Pull request by ingeknudsen**

### CODE REVIEW

File: azure-pipelines/build-and-push-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Docker image
+# Build a Docker image to deploy, run, or push to a container registry.
+# Add steps that use Docker Compose, tag images, push to a registry, run an image, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
```
Comment (nemzes) [inline azure-pipelines/build-and-push-pipeline.yml]: Should remove this generic comment or, even better, replace with a short description of the purpose of this pipeline.

File: azure-pipelines/build-only-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Docker image
+# Build a Docker image to deploy, run, or push to a container registry.
+# Add steps that use Docker Compose, tag images, push to a registry, run an image, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
```
Comment (nemzes) [inline azure-pipelines/build-only-pipeline.yml]: As above

File: azure-pipelines/build-and-push-pipeline.yml (hunk 1 above)
Comment (ingeknudsen) [reply azure-pipelines/build-and-push-pipeline.yml]: Have provided a new description

File: azure-pipelines/build-only-pipeline.yml (hunk 2 above)
Comment (ingeknudsen) [reply azure-pipelines/build-only-pipeline.yml]: Have provided a new description

**Pull request merged**